onboard/catkin_ws/src/task_planning/scripts/gate_task.py

```diff
@@ -0,0 +1,337 @@
+#!/usr/bin/env python
```

Comment:
This still exists, but heavily modified. Maybe the state machine creation itself should be recovered? Need to talk to someone who was at competition